clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForDsn.ts

```diff
@@ -52,14 +52,14 @@ export class CopybookDownloaderForDsn extends ZoweExplorerDownloader {
     copybookName: CopybookName,
     documentUri: string,
     dsnPath: string,
+    profile?: string,
   ): Promise<boolean> {
-    const providedProfile = ProfileUtils.getProfileNameForCopybook(
-      documentUri,
-      this.explorerAPI,
-    );
+    const providedProfile = profile
+      ? profile
+      : ProfileUtils.getProfileNameForCopybook(documentUri, this.explorerAPI);
```

Comment:
At this point just make the profile mandatory and move `ProfileUtils.getProfileNameForCopybook` to the (only) call site.